### PR TITLE
Flake8 phase1

### PR DESCRIPTION
**Pull request by jochman**
fixes: https://app.circleci.com/pipelines/github/demisto/content/14986/workflows/31aa6a93-61b0-45de-b522-7406e78a6fb2/jobs/56786/parallel-runs/1?filterBy=FAILED

connected: https://github.com/demisto/content/pull/7338
- [x] Ready
FeedMitreAttack
ImageOCR
MicrosoftGraphListener
Shodan_V2

### CODE REVIEW

File: Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.yml [hunk 1]
```diff
@@ -137,8 +137,10 @@ script:
       description: Indicator ID.
       type: number
     description: Lookup reputation in the indicators.
-  dockerimage: demisto/taxii2:1.0.0.6793
+  dockerimage45: demisto/taxii2:1.0.0.6793
+  dockerimage: demisto/taxii2:1.0.0.8854
   feed: true
+
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.yml:143]: remove empty line

File: Packs/FeedMitreAttack/ReleaseNotes/1_0_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### MITRE ATT&CK
+- Formatting fixes.
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_0_3.md:4]: i think we can keep empty rn here

File: Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml [hunk 3]
```diff
@@ -385,7 +385,8 @@ script:
     description: Tests connectivity of the email.
     execution: false
     name: msgraph-mail-test
-  dockerimage: demisto/crypto:1.0.0.303
+  dockerimage45: demisto/crypto:1.0.0.303
```
Comment (Itay4) [inline Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml:388]: why do we need 45 here?

File: Packs/Shodan/Integrations/Shodan_v2/Shodan_v2.yml [hunk 4]
```diff
@@ -380,7 +380,8 @@ script:
     description: Resumes receiving notifications for network alerts that are set off by the specified triggers.
     execution: false
     name: shodan-network-alert-remove-service-from-whitelist
-  dockerimage: demisto/python3:3.7.3.286
+  dockerimage45: demisto/python3:3.7.3.286
```
Comment (Itay4) [inline Packs/Shodan/Integrations/Shodan_v2/Shodan_v2.yml:383]: why is 45 needed here?

File: Packs/ThreatQ/Integrations/ThreatQ_v2/ThreatQ_v2.yml [hunk 5]
```diff
@@ -2198,7 +2198,8 @@ script:
         Registry Key, Service Name, SHA-1, SHA-256, SHA-384, SHA-512, String, x509 Serial, x509 Subject,
         URL, URL Path, User-agent, Username, X-Mailer"
     description: Runs an advanced indicator search.
-  dockerimage: demisto/python3:3.8.2.6981
+  dockerimage45: demisto/python3:3.8.2.6981
```
Comment (Itay4) [inline Packs/ThreatQ/Integrations/ThreatQ_v2/ThreatQ_v2.yml:2201]: why is 45 needed here?

**Comment by Itay4** [comment]
lets work with @kirbles19 on the release notes

File: Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.yml (hunk 1 above)
Comment (Itay4) [reply Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.yml:142]: remove empty line

**Comment by Itay4** [comment]
check why mitre test fails

**Comment by jochman** [comment]
Failed on Phising (mailbox is full). We can merge @Itay4

File: Packs/FeedMitreAttack/ReleaseNotes/1_0_4.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_0_4.md:2]: lets comment this out

**Comment by Itay4** [comment]
@jochman lets comment out all rn here

**Comment by jochman** [comment]
> @jochman lets comment out all rn here

Done

**Comment by Itay4** [comment]
build fails on non related tests
merging

**Pull request merged**